explorer/scripts/build.sh

```diff
@@ -4,7 +4,7 @@ DIR="./site-config/$1/images/favicons/"
 PUBLIC_DIR="./public/favicons/"
 # init
 
-cp ./site-config/$1/prod/.env .env.production
+cp ./site-config/"$1"/"${data_browser_build_env:-prod}"/.env .env.production
```

Comment:
I'm not proud of this, but didn't see any other way. It seems like the "dev" and "build" concepts collide with the fact that we need two forms of a "build", one for `anvildev` and one for `anvilprod`. We can't do a "dev" build for `anvildev`. Both must be "builds".